nebula-importer.yaml

```diff
@@ -19,6 +19,10 @@ clientSettings:
       USE shareholding;
       CREATE TAG person(name string);
       CREATE TAG corp(name string);
+      Create TAG INDEX person_name on person(name(8))
```

Comment:
- Could you please keep `CREATE` in the upper case?
- Please use longer length for names rather than 8